server/apolloServer.js

```diff
@@ -9,6 +11,11 @@ function newApolloServer (app, schema, httpServer, tracing, playgroundConfig, gr
 
   let apolloServer = new ApolloServer({
     schema,
+    introspection: serverSecurity.introspection,
+    validationRules: [
+      depthLimit(serverSecurity.depthLimit),
```

Comment:
What happens if depthLimit is not a valid number?

Reply:
In the npm module source code it is wrapped in a try catch block where the try returns a validationContext. If there is an error returned by the catch instead then it gets ignored in this array by apollo and by graphql as well I am assuming.

TLDR: no depth validation but nothing breaks